cmd/zdb/zdb.c

```diff
@@ -4280,6 +4554,25 @@ zdb_count_block(zdb_cb_t *zcb, zilog_t *zilog, const blkptr_t *bp,
 		    [BPE_GET_PSIZE(bp)]++;
 		return;
 	}
+	{
```

Comment:
c99 is allowed so there's no need to put this block in `{ }`.  You can remove the `{ }` and extra level of indentation.